### PR TITLE
LTD-2949-Unskip-Update-BDD

### PR DESCRIPTION
**Pull request by seijihg**
Unskipping tests and added extra steps for TAU sections.

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/LTD-2949
- [x] Jira ticket has the correct status.

### Reviewing help

- [ ] Explains how to test locally, including how to set up appropriate data
- [ ] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [ ] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [ ] Upgraded any vulnerable dependencies.
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [ ] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: ui_tests/caseworker/features/give_advice.feature [hunk 1]
```diff
@@ -50,10 +50,17 @@ Feature: I want to record my user advice and any comments and conditions relatin
...
-    Then I see the case status is now "Under review"
-    And I see the case is assigned to queues "Licensing Unit Pre-circulation Cases to Review"
+    When I go to my profile page
+    And I change my team to "Technical Assessment Unit" and default queue to "Technical Assessment Unit SIELs to Review"
+    And I go to my case list
+    Then I see previously created application
+    When I click on the application previously created
+    Then I click on Product Assessment
+    Then I select good
+    Then I add exporter suggestion with this "ML1a"
```
Comment (currycoder) [inline ui_tests/caseworker/features/give_advice.feature:60]: I don't think this line makes much sense to read anymore.  Isn't it something like "I select the CLE ML1a"?

File: ui_tests/caseworker/features/routing_rules.feature [hunk 2]
```diff
@@ -19,10 +19,17 @@ Feature: I want to have cases be automatically routed to relevant work queues an
...
-    Then I see the case status is now "Under review"
-    And I see the case is assigned to queues "Licensing Unit Pre-circulation Cases to Review"
+    When I go to my profile page
+    And I change my team to "Technical Assessment Unit" and default queue to "Technical Assessment Unit SIELs to Review"
+    And I go to my case list
+    Then I see previously created application
+    When I click on the application previously created
+    Then I click on Product Assessment
+    Then I select good
+    Then I add exporter suggestion with this "ML1a"
```
Comment (currycoder) [inline ui_tests/caseworker/features/routing_rules.feature:29]: Same comment about reformatting this line a bit

File: ui_tests/caseworker/features/routing_rules.feature [hunk 3]
```diff
@@ -19,10 +19,17 @@ Feature: I want to have cases be automatically routed to relevant work queues an
...
+    Then I select report summary and regime to none and submit
```
Comment (currycoder) [inline ui_tests/caseworker/features/routing_rules.feature:30]: Same comment about "And"

File: ui_tests/caseworker/features/give_advice.feature [hunk 4]
```diff
@@ -50,10 +50,17 @@ Feature: I want to record my user advice and any comments and conditions relatin
...
+    Then I select report summary and regime to none and submit
```
Comment (currycoder) [inline ui_tests/caseworker/features/give_advice.feature:61]: All of these lines can start "And" rather than "Then"

**Review by currycoder — COMMENTED** [review]
Nice work, just a few minor comments inline

File: ui_tests/caseworker/features/routing_rules.feature (hunk 3 above)
Comment (seijihg) [reply ui_tests/caseworker/features/routing_rules.feature:30]: what is the difference between "and" and "then"?

Comment (currycoder) [reply ui_tests/caseworker/features/routing_rules.feature]: Just that it's more readable as a testcase.  "Then this happens And this happens And this happens"

**Review by currycoder — APPROVED** [review]

**Pull request merged**